src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/activitiesappointments/ActivitiesAppointmentsClient.kt

```diff
@@ -61,13 +61,13 @@ class ActivitiesAppointmentsClient(private val activitiesAppointmentsApiWebClien
           tierCode = AppointmentSeriesCreateRequest.TierCode.TIER_1,
           inCell = false,
           startDate = startDate,
-          startTime = startTime.toIsoTime(),
-          endTime = endTime.toIsoTime(),
+          startTime = startTime.toHourMinuteStyle(),
+          endTime = endTime.toHourMinuteStyle(),
           internalLocationId = internalLocationId,
+          extraInformation = extraInformation,
```

Comment:
This was also mandatory for VLB bookings.  It's free text so open to suggestions if you want it to be different.